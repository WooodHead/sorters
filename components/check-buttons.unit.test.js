import CheckButtons from './check-buttons'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('check-buttons', () => {
    it('shallow-renders', () => {
        const component = shallow(<CheckButtons
            id="buttons"
            label="Buttons"
            values={{
                foo: {
                    label: 'Foo',
                },
                bar: {
                    label: 'Bar',
                    default: true,
                },
            }}
        />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
