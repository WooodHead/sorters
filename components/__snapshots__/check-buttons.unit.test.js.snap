// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check-buttons shallow-renders 1`] = `
<div
  className="form-group"
>
  <label>
    Buttons
  </label>
  <div
    className="checkbox"
  >
    <label>
      <input
        id="buttons"
        name="buttons"
        type="checkbox"
        value="foo"
      />
       
      Foo
    </label>
  </div>
  <div
    className="checkbox"
  >
    <label>
      <input
        defaultChecked={true}
        id="buttons"
        name="buttons"
        type="checkbox"
        value="bar"
      />
       
      Bar
    </label>
  </div>
</div>
`;
